.github/workflows/tests_trigger.yml

```diff
@@ -0,0 +1,28 @@
+jobs:
+    liferay_docker_tests:
+        env:
+            LIFERAY_RELEASE_GITHUB_PAT: ${{ secrets.LIFERAY_RELEASE_GITHUB_PAT }}
+        if: (github.repository == 'natocesarrego/liferay-docker')
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+            - name: Running test_tag_release test.
+              run: |
+                cd release
+
+                output=$(./test_tag_release.sh)
+
+                echo "$output"
+
+                if [[ "$output" == *"FAILED"* ]]
+                then
+                    echo "Test failed. Exiting with status 1."
+
+                    exit 1
+                fi
```

Comment:
@lucasmiranda0, let's prepare it to run all existing tests. I was thinking in something like:
```
run: |
    cd release

    ./run_tests_release.sh
```
Then you can create a new file which should be placed inside the folder `release` and should have the following content:
```
#!/bin/bash

function main {
    find . -name "test_*.sh" -exec ./{} \;
}

main
```
This way we'll always run all existing tests for sure, even the new ones recently created, and if we desire to do any change we just need to modify the file `run_tests_release.sh` instead of modify the GitHub Action. :smile: 

Does it make sense to you?